app/views/multiyear/do_you_live_in_scotland.scala.html

```diff
@@ -43,7 +43,7 @@ <h1 class="heading-xlarge">@Html(Messages("eligibility.check.do.you.live.in.scot
         @templates.input_radio_group(
         doYouLiveInScotlandForm("do-you-live-in-scotland"),
         Seq(("true", Messages("radio.yes")), ("false", Messages("radio.no"))),
-        '_legend -> Html(Messages("eligibility.check.do.you.live.in.scotland.legend")),
+        '_legend -> Html(Messages("eligibility.check.do.you.live.in.scotland.h1")),
```

Comment:
Use implicit messages
```suggestion
        '_legend -> Html(messages("eligibility.check.do.you.live.in.scotland.h1")),
```